docs/Configuration.rst

```diff
@@ -5981,6 +6036,163 @@ data is stored in the database.
 To disable the login history data storage in the database, simply set the parameter 
 ``exo.audit.login.enabled`` to ``false``.
 
+.. _Configuration.Wallet:
+
+================
+eXo Wallet addon
+================
+
+The eXo Wallet addon use `Ethereum Blockchain <https://www.ethereum.org/>`__ to reward users based on a crypto-currency (*Cauri*) created inside this blockchain.
+
+Blockchain network settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addon is configured by default to communicate with Ethereum Mainnet throught `Infura <https://infura.io/>`__ using websocket and http protocols:
+
+.. code-block:: jproperties
+
+    exo.wallet.blockchain.networkId=1
+    exo.wallet.blockchain.network.websocket=wss://mainnet.infura.io/ws/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+    exo.wallet.blockchain.network.http=https://mainnet.infura.io/v3/a1ac85aea9ce4be88e9e87dad7c01d40
+
+- The Websocket endpoint is used to:
+
+   -  Scan new mined blocks on blockchain to verify if there is a new mined transaction that involves a wallet of a recognized user.
+      The periodicity can be configured using the following property (Default value: every hour):
+
+    .. code-block:: jproperties
+
+          exo.wallet.ContractTransactionVerifierJob.expression=0 0 * ? * * *
+
+   -  Checks all pending transactions sent using wallet application to update its state internally in internal store and to send notifications for receiver and sender.
+      The check is made periodically and can be configured using the following property (Default value: every 10 seconds):
+
+    .. code-block:: jproperties
+
+          exo.wallet.PendingTransactionVerifierJob.expression=0/10 * * * * ?
+
+   -  Send transactions on blockchain using *Admin wallet* that is used to administrate all other wallets.
+      In fact this wallet is used to initialize and send rewards to users and spaces wallets.
+
+- The HTTP endpoint is used by *users and spaces wallets* using `Web3.js <https://web3js.readthedocs.io/>`__ (Browser side) to communicate with the blockchain to send transactions, unlike the *Admin wallet* which uses `Web3j <https://web3j.io/>`__ (Server side) to send transactions.
+
+// TODO ajouter un schéma avec la blockchain, la UI d'eXo Wallet et le server eXo derrière qui communiquent ensemble (A DEMANDER AU DESIGNER de l'aide !!!!)
+
+Blockchain transaction settings
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- When sending a transaction to Ethereum blockchain, some generic properties are required, especially *gas limit* and *gas price*.
+
+  - **Gas limit**
+
+    The *gas limit* is the maximum gas that can take a transaction using *Cauri* contract. It can be modified using the following property:
```

Comment:
```suggestion
    The *gas limit* is the maximum gas that can take a transaction using the token contract. It can be modified using the following property:
```
perhaps we could have an external link to a definition of the term gas

Reply:
Done